public/vendors.csv

```diff
@@ -0,0 +1,4 @@
+business_name,phone,email
```

Comment:
Since we allow them to put in business name, contact name, phone, email, address, we should let them populate all those fields in the csv.